### PR TITLE
Don't use TEC functions in ET unless we check for existence first

### PR DESCRIPTION
**Pull request by borkweb**
See: https://central.tri.be/issues/72499

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$event' | [email.php:340](https://github.com/moderntribe/event-tickets/blob/3de5b29fcb4e45c9462c1f0c4896ba444148361a/src/views/tickets/email.php#L340)

> via jenkins/codesniffer-phpcs

File: src/template-tags/tickets.php [hunk 1]
```diff
@@ -10,6 +10,46 @@
 	exit( '-1' );
 }
 
+if ( ! function_exists( 'tribe_tickets_parent_post' ) ) {
+	/**
+	 * Returns the current post object that can have tickets attached to it
+	 *
+	 * Optionally the post object or ID of a post can be passed in and,
```
Comment (barryhughes) [inline src/template-tags/tickets.php]: Might it be clearer to write, _the post object or ID of a **ticket** post can be passed in and, again, the **parent (event)** post object will be returned if possible._

Though I'm pretty sure lots of us will get the gist, it's slightly ambiguous as currently worded.

Comment (borkweb) [inline src/template-tags/tickets.php]: Yeah. I was being lazy and copy/pasting the comment from the-events-calendar and removing the word *event*. I'll use your suggestion!

Comment (borkweb) [inline src/template-tags/tickets.php]: Adjusted!

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$event' | [email.php:340](https://github.com/moderntribe/event-tickets/blob/ab8c4b1183660bb9a4a6fcd0e2dde79e42c1b406/src/views/tickets/email.php#L340)

> via jenkins/codesniffer-phpcs

**Comment by barryhughes** [comment]
For this ticket/PR are we also interested in usage of `tribe_get_start_date()`? [(Example here.)](https://github.com/moderntribe/event-tickets/blob/develop/src/views/tickets/email-tickets-moved.php#L113)

**Comment by borkweb** [comment]
Oh, for sure! I'll address that one too.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$event' | [email.php:340](https://github.com/moderntribe/event-tickets/blob/4512c9e0529f9045f97d6432f1b050ebb50acaa0/src/views/tickets/email.php#L340)

> via jenkins/codesniffer-phpcs

**Comment by barryhughes** [comment]
🌟

**Pull request merged**